app/src/main/res/values/strings.xml

```diff
@@ -53,18 +53,32 @@
     <string name="haushaltlöschenConfirm">Sind sie sicher, dass sie den Hauhalt löschen wollen?</string>
 
     <string name="amortPV_hint_leistung">z.B. 4.5</string>
-    <string name="amortPV_hint_anschaffung">z.B. 6525</string>
-    <string name="amortPV_hint_jahresertragkWh_zahl">z.B. 4095</string>
-    <string name="amortPV_hint_amortdauer_zahl">z.B. 17.6</string>
-    <string name="amortPV_hint_jahresertragEuro_zahl">z.B. 369.78</string>
-    <string name="amortPV_text_beschreibung">Vereinfachte Bestimmung der Amortisationszeit für eine Photovoltaik-Anlage</string>
+    <string name="amortPV_hint_anschaffung">z.B. 10000</string>
+    <string name="amortPV_hint_jahresertragkWh_zahl">z.B. 4500</string>
+    <string name="amortPV_hint_amortdauer_zahl">z.B. 13.110</string>
+    <string name="amortPV_hint_jahresertragEuro_zahl">z.B. 406.35</string>
+    <string name="amortPV_hint_kwh_pro_kwp">z.B. 1000</string>
+    <string name="amortPV_hint_verguetung">z. B. 9.03</string>
+    <string name="amortPV_hint_eigenverbrauch">z. B. 30%</string>
+    <string name="amortPV_hint_preis_kwh">z. B. 26.40</string>
+    <string name="amortPV_edit_kwh_pro_kwp">1000</string>
+    <string name="amortPV_edit_verguetung">9.03</string>
+    <string name="amortPV_edit_eigenverbrauch">30</string>
+    <string name="amortPV_edit_preis_kwh">26.4</string>
+    <string name="amortPV_text_beschreibung">Bestimmung der Amortisationszeit für eine Photovoltaik-Anlage</string>
     <string name="amortPV_text_leistung">Leistung der PV-Anlage in kWp</string>
     <string name="amortPV_text_anschaffung">Anschaffungskosten in Euro</string>
     <string name="amortPV_text_jahresertragkWh">Jahresertrag in kWh</string>
-    <string name="amortPV_text_jahresertragkWh_beschreibung">(basierend auf durchschnittlichem Ertrag in Deutschland von 910kWh pro 1kWp im Jahr, Stand: Juli 2020)</string>
     <string name="amortPV_text_amortdauer">Amortisationsdauer in Jahren</string>
-    <string name="amortPV_text_amortdauer_beschreibung">(basierend auf einer Vergütung von 9.03 Cent/kWh, Stand: Juli 2020)</string>
     <string name="amortPV_text_jahresertragEuro">Jahresertrag in Euro</string>
+    <string name="amortPV_text_kwh_pro_kwp">Ertrag pro kWp in kWh</string>
+    <string name="amortPV_text_ertrag_beschreibung">(Der Ertrag pro kWp gibt an, wieviel kWh die PV-Anlage im Durchschnitt pro kWp (Nennleistung) im Jahr produziert.
+        Im Süden Deutschlands beträgt dieser ca. 1020 und im Norden ca. 900 kWh/kWp)</string>
```

Comment:
Find ich gut,dass das so genau ist, da dadurch leute sich besser vorstellen können, was genau gemacht werden muss, ohne davon genau zu wissen. Und auch gut, dass du das in die Strings hast und nicht hard gecoded.